src/views/api-view.js

```diff
@@ -164,10 +166,18 @@ class ApiView extends ViewMixin(ReduxMixin(PolymerElement)) {
                 type: Boolean,
                 statePath: 'app.drawerOpened',
                 observer: '_toggleDrawer'
+            },
+            api:{
+                type: Boolean,
+                computed: 'computeApi(subpage, data)'
             }
         }
     }
 
+    computeApi(pageName, data) {
+        if(pageName && data) return !data.pages[pageName].noApi;
+    }
+
```

Comment:
1)
You have two return types here. `bool` and well.... an implicit `undefined`.
I think it would be more obvious to see the purpose of this function's result if it used explicit `return`s for all possibilities.
I'd go for something like `return pageName && data && !data.pages[pageName].noApi` so the return type is always a `bool`.

2)
Plus, I think the function's name could be more expressive. For example `hasApiTab`.